src/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit.csproj

```diff
@@ -68,8 +101,16 @@
     </PackageReference>
   </ItemGroup>
   
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)Esri.ArcGISRuntime.Toolkit.Android.targets">
+      <Pack>true</Pack>
+      <PackagePath>build\$(AndroidTargetFramework)\</PackagePath>
+    </None>
+  </ItemGroup>
+
   <ItemGroup>
     <_ResxFiles Remove="**\*.resx" />
+    <None Remove="**\*.axml" />
```

Comment:
Remove. The axml files should be in the shared project

Reply:
Doesn't work, as mentioned in my previous comment.  The files need to be under the library project's directory for things to work.